### ISSUE TITLE
kubernetes_sumologic plugin always adds a timestamp field to json logs

### ISSUE THREAD

**Issue by darend** [SumoLogic/fluentd-kubernetes-sumologic#61]
## what

kubernetes_sumologic plugin always adds a timestamp field to json logs. This can lead to invalid json when the application output already includes a `timestamp` field.

See https://github.com/SumoLogic/fluentd-output-sumologic/blob/034168eaa280d79424fa0d90ee6668f10ec218fe/lib/fluent/plugin/out_sumologic.rb#L206-L214

## reproduce

Application outputs a log message like:

```
{"timestamp":"2018-05-02T16:06:52.550+0000","severity":"info","message":"hello world"}
```

The ingested message will become (with `json_merge`):
```
{"time":1525276647680,"timestamp": "2018-05-02T15:57:27.680+0000", "timestamp":"2018-05-02T16:06:52.550+0000","severity":"info","message":"hello world"}
```

## expected

Possibilities:

- If the output already includes the `timestamp` key do not add one. 
- allow the timestamp field injected by the plugin to be configurable
- allow the timestamp field injected by the plugin to be disabled

**Comment by darend** [comment]
This was fixed in the plugin: https://github.com/SumoLogic/fluentd-output-sumologic/pull/27

**Comment by frankreno** [comment]
This has been fixed in v1.14, ADD_TIMESTAMP environment variable defaults to `true` for backward compatibility.  PR for helm chart updates submitted and is just pending approval - https://github.com/kubernetes/charts/pull/5338
